### DISCUSSION TITLE
oracle cdc：The connector is now using the maximum batch size 100000 when querying the LogMiner view. This could be indicative of large SCN gaps

### DISCUSSION THREAD

**Discussion by MonsterChenzhuo** [apache/flink-cdc#1430] [Q&A]
taskmanage不断答疑以下内容，并且新的数据不在摄入，整个任务呈现卡死状态：
The connector is now using the maximum batch size 100000 when querying the LogMiner view. This could be indicative of large SCN gaps
求教该如何解决

**Comment by spoon-lz** [comment]
我也遇到了相同的问题，请问你是如何解决的呢

**Comment by li57598** [comment]
遇到同样的问题，你们都怎么解决的

**Comment by LiaoMinYu** [comment]
遇到同样的问题，你们都怎么解决的

**Reply by li57598** [reply]
无解呀，oracle cdc太难用了

**Reply by LiaoMinYu** [reply]
大佬有代替方案吗

**Comment by thachnv92** [comment]
This document may help us!
https://debezium.io/documentation/reference/stable/connectors/oracle.html#scn-jumps
Read it and re-config connector.

**Comment by thachnv92** [comment]
Can you try with new config:

        "log.mining.batch.size.default": "100000",
        "log.mining.batch.size.min":"10000",
        "log.mining.batch.size.max":"1000000",
        "log.mining.sleep.time.default":"200",
        "log.mining.sleep.time.min":"0",
        "log.mining.sleep.time.max":"1000",
        "log.mining.sleep.time.increment":"100"

**Reply by LiaoMinYu** [reply]
I tried what you said last night, but I found the same error today

**Reply by LiaoMinYu** [reply]
<img width="1133" alt="image" src="https://user-images.githubusercontent.com/40682369/233241442-b1665017-35b2-4157-adc9-0eb831f8984c.png">
<img width="716" alt="image" src="https://user-images.githubusercontent.com/40682369/233241517-6c298a0e-1420-477a-b2cd-e43c6bf25a1f.png">

**Reply by thachnv92** [reply]
Maybe, your DB is so much update, so gap of SCN is large.
Try to increase log.mining.batch.size.max and find reason value.
EX: 2000000

**Reply by LiaoMinYu** [reply]
It's impossible！Because I'm the only one using the database，I didn't change the data

**Comment by ZwzYJ999** [comment]
我也遇到了相同的问题, 请问有解决吗

**Comment by zhangfqcxl** [comment]
log.mining.batch.size.min=200000
log.mining.batch.size.max=50000000
尽量设置大点
